### PR TITLE
 JENKINS-59513 - Fixed defunct hyperlink in PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by agmt5989**
Removed a defunct hyperlink and placed an intuitive directive in its place  [JENKINS-59513](https://issues.jenkins-ci.org/browse/JENKINS-59513).

### Proposed changelog entries

* None

### Description

Replaced a defunct hyperlink in the Pull Request Template file with a simple, human-readable directive, as described in issue [JENKINS-59513](https://issues.jenkins-ci.org/browse/JENKINS-59513).

### Submitter checklist

- [x] JIRA issue is well described
- [x] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers
@oleg-nenashev 

This change has no autotests because none is necessary.

### CODE REVIEW

**Review by jvz — CHANGES_REQUESTED** [review]
The fake link is intentional here. You're supposed to replace `XXXXX` with the actual Jira issue id when you submit the PR.

**Comment by agmt5989** [comment]
@jvz oh! I didn't realize. May I close this PR?

**Comment by agmt5989** [comment]
I might suggest that a top-level comment be placed to clarify that intention.

**Comment by jvz** [comment]
Feel free to update this PR to add that comment. I wonder if we can use some sort of github hook to verify that the PR doesn't include that link?

**Comment by agmt5989** [comment]
Done, although I don't understand what github hook can be used to verify that.

**Review by jvz — APPROVED** [review]

**Comment by jvz** [comment]
Yeah, I have no idea, either. Could be neat, though. :)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<!-- Comment: 
+A great PR typically begins with the line below.
+Replace XXXXX with the numeric part of the issue's id you created on JIRA.
```
Comment (res0nance) [inline .github/PULL_REQUEST_TEMPLATE.md]: Perhaps make a mention that a jira-ticket is required for backporting

Comment (agmt5989) [reply .github/PULL_REQUEST_TEMPLATE.md]: > 
> 
> Perhaps make a mention that a jira-ticket is required for backporting

Done.

**Review by oleg-nenashev — APPROVED** [review]
Thanks @agmt5989 !

**Comment by oleg-nenashev** [comment]
Merging without timeout since it is not a production code. We can apply changes in follow-ups if needed

**Review by res0nance — APPROVED** [review]

**Comment by agmt5989** [comment]
My pleasure.
I'm not sure why the failing tests.

**Review by timja — APPROVED** [review]

**Comment by agmt5989** [comment]
I'm ready to close this.

**Comment by oleg-nenashev** [comment]
There is no way it breaks CI on its own, just some INFRA instability.
Will merge as is

**Comment by agmt5989** [comment]
Thank you.

**Pull request merged**